app/templates/_package.json

```diff
@@ -14,7 +14,9 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo <% if (babel) { %>src/*.js test*.js && npm run babel <% } %>&& <% if (nyc) { %>nyc <% } %>ava"<% if (babel) { %>,
+    "babel": "babel src --out-dir=. --presets=es2015",
```

Comment:
[pageres](https://github.com/sindresorhus/pageres) uses `dist` a well. It's easier to ignore those files for git as well because you only have to add `dist` instead of individual files. So `.gitignore` should be adjusted as well when babel is used.